src/Microsoft.Identity.Client/ITokenCacheInternal.cs

```diff
@@ -58,10 +58,10 @@ internal interface ITokenCacheInternal : ITokenCache
         ILegacyCachePersistence LegacyPersistence { get; }
         ITokenCacheAccessor Accessor { get; }
         void RemoveMsalAccount(IAccount account, RequestContext requestContext);
-        ICollection<string> GetAllAccessTokenCacheItems(RequestContext requestContext);
-        ICollection<string> GetAllRefreshTokenCacheItems(RequestContext requestContext);
-        ICollection<string> GetAllIdTokenCacheItems(RequestContext requestContext);
-        ICollection<string> GetAllAccountCacheItems(RequestContext requestContext);
+        ICollection<MsalAccessTokenCacheItem> GetAllAccessTokenCacheItems(RequestContext requestContext);
```

Comment:
nit: might be a good time to push RequestContext as part of the state instead of it being in each call.

Reply:
Right now, the cacahe is global to the ClientApplication so the requestcontext needs to be sent in to each call.  In C++, the cache manager is built per-request so it can get away without having this.  I'll see if there's a way to apply that pattern here so we don't have to keep sending it around in the api.